<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>TransactionDialog</class>
	<widget class="QDialog" name="TransactionDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>400</width>
				<height>350</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Transaktion</string>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<item>
				<widget class="QGroupBox" name="typeGroupBox">
					<property name="title">
						<string>Transaktionstyp</string>
					</property>
					<layout class="QHBoxLayout" name="horizontalLayout">
						<item>
							<widget class="QRadioButton" name="incomeRadioButton">
								<property name="text">
									<string>Einnahme</string>
								</property>
								<property name="checked">
									<bool>false</bool>
								</property>
							</widget>
						</item>
						<item>
							<widget class="QRadioButton" name="expenseRadioButton">
								<property name="text">
									<string>Ausgabe</string>
								</property>
								<property name="checked">
									<bool>true</bool>
								</property>
							</widget>
						</item>
					</layout>
				</widget>
			</item>
			<item>
				<layout class="QFormLayout" name="formLayout">
					<item row="0" column="0">
						<widget class="QLabel" name="dateLabel">
							<property name="text">
								<string>Datum:</string>
							</property>
						</widget>
					</item>
					<item row="0" column="1">
						<widget class="QDateEdit" name="dateEdit">
							<property name="calendarPopup">
								<bool>true</bool>
							</property>
						</widget>
					</item>
					<item row="1" column="0">
						<widget class="QLabel" name="amountLabel">
							<property name="text">
								<string>Betrag (€):</string>
							</property>
						</widget>
					</item>
					<item row="1" column="1">
						<widget class="QDoubleSpinBox" name="amountSpinBox">
							<property name="alignment">
								<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
							</property>
							<property name="suffix">
								<string> €</string>
							</property>
							<property name="decimals">
								<number>2</number>
							</property>
							<property name="maximum">
								<double>1000000.000000000000000</double>
							</property>
						</widget>
					</item>
					<item row="2" column="0">
						<widget class="QLabel" name="categoryLabel">
							<property name="text">
								<string>Kategorie:</string>
							</property>
						</widget>
					</item>
					<item row="2" column="1">
						<widget class="QComboBox" name="categoryComboBox">
							<property name="editable">
								<bool>true</bool>
							</property>
						</widget>
					</item>
					<item row="3" column="0">
						<widget class="QLabel" name="descriptionLabel">
							<property name="text">
								<string>Beschreibung:</string>
							</property>
						</widget>
					</item>
					<item row="3" column="1">
						<widget class="QTextEdit" name="descriptionTextEdit">
							<property name="maximumSize">
								<size>
									<width>16777215</width>
									<height>100</height>
								</size>
							</property>
						</widget>
					</item>
				</layout>
			</item>
			<item>
				<spacer name="verticalSpacer">
					<property name="orientation">
						<enum>Qt::Vertical</enum>
					</property>
					<property name="sizeHint" stdset="0">
						<size>
							<width>20</width>
							<height>40</height>
						</size>
					</property>
				</spacer>
			</item>
			<item>
				<widget class="QDialogButtonBox" name="buttonBox">
					<property name="orientation">
						<enum>Qt::Horizontal</enum>
					</property>
					<property name="standardButtons">
						<set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections>
		<connection>
			<sender>buttonBox</sender>
			<signal>accepted()</signal>
			<receiver>TransactionDialog</receiver>
			<slot>accept()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>248</x>
					<y>254</y>
				</hint>
				<hint type="destinationlabel">
					<x>157</x>
					<y>274</y>
				</hint>
			</hints>
		</connection>
		<connection>
			<sender>buttonBox</sender>
			<signal>rejected()</signal>
			<receiver>TransactionDialog</receiver>
			<slot>reject()</slot>
			<hints>
				<hint type="sourcelabel">
					<x>316</x>
					<y>260</y>
				</hint>
				<hint type="destinationlabel">
					<x>286</x>
					<y>274</y>
				</hint>
			</hints>
		</connection>
	</connections>
</ui>